test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -313,7 +312,11 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 
 			origBannedCPUsFile := "/etc/sysconfig/orig_irq_banned_cpus"
 			By(fmt.Sprintf("Checking content of %q on node %q", origBannedCPUsFile, node.Name))
-			expectFileEmpty(node, origBannedCPUsFile)
+			fullPath := filepath.Join("/", "rootfs", origBannedCPUsFile)
```

Comment:
is the a reason to expand the expectFileEmpty function here ?     If so then you can delete the function itself since its only used here  (unless we need to keep this function extracted for other uses)

Reply:
> since its only used here
Exactly the reason for inline the function

i'll delete the function forgot about it.